jwst/regtest/test_NIRcam_image2a.py

```diff
@@ -0,0 +1,66 @@
+""" Test for the image2 pipeline using NIRCam image data. """
+
+import os
+from glob import glob
+
+import pytest
+
+from astropy.io.fits.diff import FITSDiff
+from jwst.pipeline import Image2Pipeline
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+
+@pytest.mark.bigdata
+def test_NIRcam_image2a_cal(rtdata, fitsdiff_default_kwargs, _jail):
```

Comment:
I believe this test is a duplicate/subset of the test_NIRCam_image2a and can be safely removed.